addons/html_editor/static/tests/wysiwyg.test.js

```diff
@@ -115,7 +116,7 @@ describe("Wysiwyg Component", () => {
         const CLOSE_ENOUGH = 10;
         const { el } = await setupWysiwyg({
             iframe: true,
-            config: { content: "<p>editable text inside the iframe</p>".repeat(30) },
+            config: { content: markup("<p>editable text inside the iframe</p>".repeat(30)) },
```

Comment:
ci flag could be removed here by using `htmlJoin` and range

not tested but something like that:
```js
htmlJoin(range(30).map(() => markup`<p>editable text inside the iframe</p>`))
```